testutil/keeper/airdrop.go

```diff
@@ -0,0 +1,53 @@
+package keeper
```

Comment:
were you planning on using this testutil module? We don't really use it anymore so I think you can just scratch it unless you had other ideas!